sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -627,6 +635,23 @@ func parseImageURISpecialRegions(input string) (ecr.ECRSpec, error) {
 		return ecr.ECRSpec{}, errors.New("incomplete reference provided")
 	}
 
+	// Return early if the FIPS endpoint is being used. amazon-ecr-containerd-resolver doesn't yet support FIPS urls:
+	// https://github.com/awslabs/amazon-ecr-containerd-resolver/blob/7b72333e780f5a5168936eae79fb89448e2f2a8f/ecr/ref.go#L43
+	// The ecr-prefix helper for admin and control host containers will have already accounted for setting this endpoint
+	// if the region has FIPS support.
+	if matches[2] == "-fips" {
+		region := matches[3]
+		_, isFips := fipsSupportedRegionMapping[region]
+		if !isFips {
+			return ecr.ECRSpec{}, fmt.Errorf("%s: %s", "invalid FIPS region in internal mapping", region)
```

Comment:
Shouldn't the error be `invalid FIPS region` or `unsupported FIPS region`? Otherwise it is confusing and makes me think that the region was invalid in the internal mapping, but in reality this error will occur when an invalid FIPS region is used.